<launch>
  <group ns="UAV1">
    <param name="source_Q"		type="double" value="13" />
    <param name="source_D"		type="double" value="0.003" />
    <param name="source_tau"		type="double" value="1200" />
    <param name="building_size"		type="double" value="1" />

    <param name="search_nx"		type="double" value="25" />
    <param name="search_ny"		type="double" value="25" />
    <param name="uav_altitude"		type="double" value="1.1" />
    <param name="extra_length"		type="double" value="0.5" />
    <param name="adaptive_move_frac"	type="double" value="5" />
    <param name="sensor_sig_multip_est"	type="double" value="0.1" />
    <param name="env_sig"		type="double" value="1" />

    <param name="max_steps"		type="int" value="100" />
    <param name="number_of_particles"	type="int" value="2000" />

    <node pkg="multi_infotaxis" type="decentralized_infotaxis_node" name="decentralized_infotaxis">
    </node>

  </group>
</launch>
